src/helpers/sendBinEmbed.ts

```diff
@@ -12,7 +12,7 @@ import { Stream } from "stream";
 
 const noop = (): undefined => undefined;
 
-const MAX_FILE_SIZE = 8e6;
+const MAX_FILE_SIZE = 8 * 1024 * 1024; // 8MiB
```

Comment:
étant donnée que c'est assez aléatoire, on met (https://www.reddit.com/r/discordapp/comments/aflp3p/the_truth_about_discord_file_upload_limits/)
```suggestion
const MAX_FILE_SIZE = 8_388_120;
```

Reply:
Outdated